storelogs/main.go

```diff
@@ -0,0 +1,84 @@
+package storelogs
+
+import (
+	"bytes"
+	"container/ring"
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+type LogsStore struct {
+	log   *ring.Ring
+	Entry *logrus.Entry
```

Comment:
entry *logrus.Entry

Reply:
I need use in here newProcessLogger(sl.Entry, keepLogLines, redactWords),